From a.beregalov@gmail.com  Tue Feb 17 17:37:27 2009
From: Alexander Beregalov <a.beregalov@gmail.com>
Date: Fri, 13 Feb 2009 17:35:32 +0300
Subject: Staging: pohmelfs: net.c: include vmalloc.h
To: zbr@ioremap.net, gregkh@suse.de
Message-ID: <20090213143532.GA7266@orion>


on Sparc64:
drivers/staging/pohmelfs/net.c:33: error: implicit declaration of function 'vmalloc'
drivers/staging/pohmelfs/net.c:42: error: implicit declaration of function 'vfree'

Signed-off-by: Alexander Beregalov <a.beregalov@gmail.com>
Acked-by: Evgeniy Polyakov <zbr@ioremap.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/pohmelfs/net.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/pohmelfs/net.c
+++ b/drivers/staging/pohmelfs/net.c
@@ -22,6 +22,7 @@
 #include <linux/poll.h>
 #include <linux/swap.h>
 #include <linux/syscalls.h>
+#include <linux/vmalloc.h>
 
 #include "netfs.h"
 
